### PR TITLE
Refactor tests to pass with Forge 3

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

**Comment by gastaldi** [comment]
Great job. Do you have push access to this repository?

**Comment by bartoszmajsak** [comment]
Hey... I was in the middle of reviewing..

**Comment by gastaldi** [comment]
Oops sorry about that

**Comment by bartoszmajsak** [comment]
no worries, I guess I should have add myself as a reviewer :)

**Comment by gastaldi** [comment]
For now on I'll leave the addon maintenance to you guys only ;)

File: src/test/java/test/integration/ConfigurationIntegrationTest.java [hunk 1]
```diff
@@ -1,129 +0,0 @@
...
-import org.junit.Ignore;
-import org.junit.runner.RunWith;
-
-import javax.inject.Inject;
-
-/**
- * @Author Paul Bakker - paul.bakker.nl@gmail.com
- */
-@RunWith(Arquillian.class) @Ignore
-public class ConfigurationIntegrationTest
```
Comment (bartoszmajsak) [inline src/test/java/test/integration/ConfigurationIntegrationTest.java:27]: So nothing relevant here worth covering?

**Comment by lordofthejars** [comment]
I have no rights, so I cannot assign reviewers nor merge PRs :)

**Comment by gastaldi** [comment]
I just sent you an invitation to join the team. See if that works for you

File: src/test/java/test/integration/AddArquillianCommandTestCase.java [hunk 2]
```diff
@@ -1,18 +0,0 @@
-package test.integration;
-
-import org.junit.Assert;
-import org.junit.Ignore;
-import org.junit.Test;
-
-@Ignore
-public class AddArquillianCommandTestCase {
```
Comment (bartoszmajsak) [inline src/test/java/test/integration/AddArquillianCommandTestCase.java:8]: There is no `add-arquillian` command  (or similar) anymore? I think we should cover all from `.command` package

**Comment by gastaldi** [comment]
Aw yeah I see you are a member now. Welcome! 😉

**Review by bartoszmajsak — COMMENTED** [review]
Do we cover all available commands now?

File: src/test/java/test/integration/AddArquillianCommandTestCase.java (hunk 2 above)
Comment (lordofthejars) [reply src/test/java/test/integration/AddArquillianCommandTestCase.java:8]: `AddArquillianCommand`is tested at `AddArquillianUniverseTest`

Comment (bartoszmajsak) [reply src/test/java/test/integration/AddArquillianCommandTestCase.java:8]: Yeah but that's it. And the test can get some love :)

File: src/test/java/test/integration/ConfigurationIntegrationTest.java (hunk 1 above)
Comment (lordofthejars) [reply src/test/java/test/integration/ConfigurationIntegrationTest.java:27]: Humm right, I deleted wrongly, because there are some things to cover.

Comment (lordofthejars) [reply src/test/java/test/integration/ConfigurationIntegrationTest.java:27]: I added in another PR

File: src/test/java/test/integration/AddArquillianCommandTestCase.java (hunk 2 above)
Comment (lordofthejars) [reply src/test/java/test/integration/AddArquillianCommandTestCase.java:8]: This one? Why, the command is already tested.

Comment (bartoszmajsak) [reply src/test/java/test/integration/AddArquillianCommandTestCase.java:8]: No, the universe test.

**Comment by lordofthejars** [comment]
@bartoszmajsak yes all commands are covered

**Comment by bartoszmajsak** [comment]
@gastaldi why https://travis-ci.org/forge/addon-arquillian/builds/188769790 is not visible in the PR?

**Comment by gastaldi** [comment]
Good question, I have no clue. I think I'll add it to forge.ci.cloudbees.com instead. Thoughts?

**Comment by bartoszmajsak** [comment]
Travis is way easier to maintain - but it's up to your org how you wanna deal with it. Probably having it unified is the easiest.

**Comment by gastaldi** [comment]
I'm fine with either choice. I was adopting Travis for the community addons and the CBees for the core addons.

Also, we should probably consider moving this repository to the arquillian org at some point.

**Comment by gastaldi** [comment]
Anyhow I created https://forge.ci.cloudbees.com/job/arquillian-addon/ and https://forge.ci.cloudbees.com/job/arquillian-addon-pull-requests/

I think we can discard Travis in this repository for now

**Pull request merged**